libs/service-portal/education-student-assessment/src/screens/EducationStudentAssessment/components/StudentAssessmentTable/StudentAssessmentTable.tsx

```diff
@@ -1,127 +1,190 @@
-    data: gradeMock,
-  },
-]
+    },
+  )
+  const educationExamResult =
+    data?.educationExamResult?.grades.map((res) => ({
+      title: `Samræmd könnunarpróf ${res.studentYear}. bekkur`,
+      data: [
+        {
+          subject: 'Enska',
+          serialRating: res.englishGrade?.grade,
+          abilityRating: res.englishGrade?.competence,
+          status: res.englishGrade?.competenceStatus,
+          breakdown: [
+            {
+              subject: 'Lesskilningur',
+              rate: `(${res.englishGrade?.grammar.weight}%)`,
+              serialRating: res.englishGrade?.grammar.grade,
+            },
+            {
+              subject: 'Málnotkun',
+              rate: `(${res.englishGrade?.reading.weight}%)`,
+              serialRating: res.englishGrade?.reading.grade,
+            },
+            {
+              subject: 'Framfaraeinkunn',
+              progressRating: res?.englishGrade?.progressText,
+            },
+          ],
+        },
+        {
+          subject: 'Íslenska',
+          serialRating: res.icelandicGrade?.grade,
+          abilityRating: res.icelandicGrade?.competence,
+          status: res.icelandicGrade?.competenceStatus,
+          breakdown: [
+            {
+              subject: 'Lesskilningur',
+              rate: `(${res.icelandicGrade?.grammar.weight}%)`,
+              serialRating: res.icelandicGrade?.grammar.grade,
+            },
+            {
+              subject: 'Málnotkun',
+              rate: `(${res.icelandicGrade?.reading.weight}%)`,
+              serialRating: res.icelandicGrade?.reading.grade,
+            },
+            {
+              subject: 'Framfaraeinkunn',
+              progressRating: res?.icelandicGrade?.progressText,
+            },
+          ],
+        },
+        {
+          subject: 'Stærðfræði',
+          serialRating: res.mathGrade?.grade,
+          abilityRating: res.mathGrade?.competence,
+          status: res.mathGrade?.competenceStatus,
+          breakdown: [
+            {
+              subject: 'Reikningur og aðgerðir',
+              rate: `(${res.mathGrade?.calculation.weight}%)`,
+              serialRating: res.mathGrade?.calculation.grade,
+            },
+            {
+              subject: 'Rúmfræði og mælingar',
+              rate: `(${res.mathGrade?.geometry.weight}%)`,
+              serialRating: res.mathGrade?.geometry.grade,
+            },
+            {
+              subject: 'Algebra',
+              rate: `(${res.mathGrade?.algebra.weight}%)`,
+              serialRating: res.mathGrade?.algebra.grade,
+            },
+            {
+              subject: 'Hlutföll og prósentur',
+              rate: `(${res.mathGrade?.ratiosAndPercentages.weight}%)`,
+              serialRating: res.mathGrade?.ratiosAndPercentages.grade,
+            },
+            {
+              subject: 'Framfaraeinkunn',
+              progressRating: res?.mathGrade?.progressText,
+            },
```

Comment:
We also need to add here "Orðadæmi og talnadæmi" which matches with `wordsAndNumbers`